src/main/java/mate/academy/bookshop/model/Book.java

```diff
@@ -41,4 +50,12 @@ public class Book {
 
     @Column(name = "is_deleted", nullable = false)
     private boolean isDeleted = false;
+
+    @EqualsAndHashCode.Exclude
+    @ToString.Exclude
+    @ManyToMany
+    @JoinTable(name = "books-categories",

```

Comment:
In SQL `_` is more common word separator.
```suggestion
    @JoinTable(name = "books_categories",

```